### PR TITLE
Update HOWTO-port

### PR DESCRIPTION
**Pull request by IsaacG**

### CODE REVIEW

File: HOWTO-port-an-exercise.md [hunk 1]
```diff
@@ -15,34 +15,38 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
...
    `.meta/{test.toml,version,example.sh}`
-1. rename `foo_bar.bats` to `test-foo-bar.bats`
+1. rename `${exercise/-/_}.bats` to `test-$exercise.bats`
     - note the underscore changed: the AWK track uses kebab case
-1. edit `.meta/config.js`
+1. edit `.meta/config.json`
     - fix the filenames from .sh to .awk, and the test file
     - set yourself as the author, and empty the contributors list.
-1. edit `test-foo-bar.awk` so the tests are AWK specific. 
+1. edit `test-$exercise.awk` so the tests are AWK specific.
```
Comment (glennj) [inline HOWTO-port-an-exercise.md:27]: We both missed this: this should be `test-$exercise.bats`

File: HOWTO-port-an-exercise.md [hunk 2]
```diff
@@ -15,34 +15,38 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
     - add the labels: `x:action/create`, `x:module/practice-exercise`,
       `x:status/claimed`, `x:type/content`
     - assign it to yourself.
-1. copy the exercises/practice/foo-bar directory from the bash repo.
-1. in the exercises/practice/foo-bar directory, **delete** `foo_bar.sh`, and
+1. set `exercise=foo-bar`
+1. copy the exercises/practice/$exercise directory from the bash repo.
+1. in the exercises/practice/$exercise directory, **delete** `${exercise/-/_}.sh`, and
```
Comment (glennj) [inline HOWTO-port-an-exercise.md:20]: Perhaps with backticks around the directories as well.

**Review by glennj — CHANGES_REQUESTED** [review]
Thanks for taking a close look.

File: HOWTO-port-an-exercise.md (hunk 1 above)
Comment (IsaacG) [reply HOWTO-port-an-exercise.md:27]: Done

File: HOWTO-port-an-exercise.md (hunk 2 above)
Comment (IsaacG) [reply HOWTO-port-an-exercise.md:20]: Done

File: HOWTO-port-an-exercise.md [hunk 3]
```diff
@@ -15,36 +15,57 @@ Here's a step-by-step procedure to port the **_foo-bar_** exercise:
...
-    bin/configlet sync -u -e foo-bar
-    bin/configlet lint
-    ```
-    And make sure those all pass.
-1. commit your code and run
-    ```sh
-    bin/validate_one_exercise exercises/practice/foo-bar
-    ```
-    And make sure it passes
+
```
Comment (glennj) [inline HOWTO-port-an-exercise.md:18]: This works. I'm glad you kept the original english steps as somments.

If we add another bullet point here:
```none
1. follow along with these steps to create the exercise:
```
and then indent the code block by 4 spaces, then the "push your branch" item at the bottom will render as "3. ..."

**Review by glennj — CHANGES_REQUESTED** [review]

**Review by glennj — APPROVED** [review]

**Pull request merged**